packages/fast-usdc-contract/test/snapshots/fast-usdc.contract.test.ts.md

```diff
@@ -3166,6 +3169,9 @@ Generated by [AVA](https://avajs.dev).
           operatorPowers: Object @Alleged: Fast USDC Feed operatorPowers {},
           public: Object @Alleged: Fast USDC Feed public {},
         },
+        ForwardInProgress: [
+          '0x000005',
```

Comment:
Expected? We are probably missing a `await transmitVTransferEvent('acknowledgementPacket', -1);` somewhere, but wondering if this is intentional.

Reply:
No it wasn't. I'll look for that to help make the tests independent.